modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMHomePage.testcase

```diff
@@ -38,14 +38,14 @@ definition {
 	@description = "This is a test for LPS-171642. User can access MDF Request Listing through Quick Links"
 	@priority = "4"
 	test CanAccessMDFRequestListingViaQuickLinks {
-		task ("When I click on the MDF") {
+		task ("When I click on 'MDF' button, on Quick Links section") {
 			Click(locator1 = "PRMHomePage#CLICK_MDF");
```

Comment:
This click could be a macro, since you are going to add variables in the path, and it can be used in other cases.

The macro name could be `gotoQuickLinksOption`. What do you think?